scripts/us_eia/opendata/process/test_data/nuc_status.txt

```diff
@@ -0,0 +1,5 @@
+{"series_id":"NUC_STATUS.OUT.621.D","name":"Nuclear generating capacity outage at Turkey Point, daily","units":"Megawatts","f":"D","unitsshort":"MW","description":"Nuclear generating capacity outage at Turkey Point, daily. See https:\/\/www.eia.gov\/nuclear\/outages\/.  For more information visit, Nuclear Regulatory Commission's Power Reactor Status Report","source":"Nuclear Regulatory Commisssion's Power Reactor Status Report; EIA, U.S. Energy Information Administration","iso3166":"USA-FL","lat":"25.4356","lon":"-80.3308","geography":"USA-FL","start":"20070101","end":"20210511","last_updated":"2021-05-11T10:38:50-04:00","latlon":"25.4356,-80.3308","data":[["20210511",0],["20210510",0],["20070101",0]]}
+{"series_id":"NUC_STATUS.OUT_PCT.869-2.D","name":"Nuclear generating capacity percent outage for generator 2 at Dresden Generating Station, daily","units":"percent","f":"D","unitsshort":"%","description":"Nuclear generating capacity percent outage for generator 2 at Dresden Generating Station, daily. See https:\/\/www.eia.gov\/nuclear\/outages\/.  For more information visit, Nuclear Regulatory Commission's Power Reactor Status Report","source":"Nuclear Regulatory Commisssion's Power Reactor Status Report; EIA, U.S. Energy Information Administration","iso3166":"USA-IL","lat":"41.39","lon":"-88.27","geography":"USA-IL","start":"20070101","end":"20210511","last_updated":"2021-05-11T10:38:50-04:00","latlon":"41.39,-88.27","data":[["20210511",73],["20210510",80],["20210509",75]]}
+{"series_id":"NUC_STATUS.OUT.US.D","name":"U.S. nuclear capacity outage, daily","units":"Megawatts","f":"D","unitsshort":"MW","description":"U.S. nuclear capacity outage, daily. See https:\/\/www.eia.gov\/nuclear\/outages\/.  For more information visit, Nuclear Regulatory Commission's Power Reactor Status Report","source":"Nuclear Regulatory Commisssion's Power Reactor Status Report; EIA, U.S. Energy Information Administration","iso3166":"USA","geography":"USA","start":"20070101","end":"20210511","last_updated":"2021-05-11T10:38:50-04:00","data":[["20210511",16404.767],["20210510",16960.869],["20210509",17374.955]]}
+{"category_id":"2890005","parent_category_id":"2889994","name":"Columbia Generating Station","notes":"","childseries":["NUC_STATUS.CAP.371-2.D","NUC_STATUS.CAP.371.D","NUC_STATUS.OUT.371-2.D","NUC_STATUS.OUT.371.D","NUC_STATUS.OUT_PCT.371.D"]}
```

Comment:
If Columbia Generating Station is a nuclear plant, do we not want it to be part of the category? Like how we exclude "by Area" categories in PET/NG.

Reply:
ah yes, that makes sense. i'll make the change